pages/getting-started/install-memgraph/docker-compose.mdx

```diff
@@ -3,72 +3,78 @@ title: Docker Compose
 description: Experience Memgraph's integration with Docker Compose. Our dedicated documentation walks you through maximizing your graph computing capabilities.
 ---
 
+import { Callout } from 'nextra/components'
+
 # Docker Compose
 
 If you define an application with **Docker Compose**, you can use that
 definition to run the application in CI, staging, or production environments.
 Here you can find `docker-compose.yml` files necessary to run
-[`memgraph-platform`](#docker-compose-for-memgraph-platform-image),
-[`memgraph-mage`](#docker-compose-for-memgraph-mage-image) and
-[`memgraph`](#docker-compose-for-memgraph-image) images.
-
-## Docker Compose for Memgraph Platform image
+[`memgraph-platform`](#docker-compose-for-memgraph-platform),
+[`memgraph-mage`](#docker-compose-for-memgraph-mage-image), and
+[`memgraph`](#docker-compose-for-memgraph-image) images. You can also find
```

Comment:
I would say Memgraph Platform, Memgraph MAGE and Memgraph (without images, since we're not using Memgraph Platform as an image anymore).

Reply:
Update. I've also updated the coresponding headings.